### ISSUE TITLE
Tagref ignores tags in dot files

### ISSUE THREAD

**Issue by sharmilajesupaul** [stepchowfun/tagref#93]
**Description**

I have a dot file that is not gitignored and checked into the repo. Tagref seems to ignore any tags in the dot file and acts like they don't exist. This is happening on version 1.3.0 & 1.3.1. 

```
❯ tagref --version 
Tagref 1.3.1
```

**Instructions to reproduce the bug**
Add a dot file to the repo containing a tag: 

```
// .happo.js
[tag: foo]
```

add a reference to the tag in a file with a name that doesn't begin with a dot
```
// cypress.json
[ref: foo]
```

this will result in the following error during lint:

```
❯ tagref                    
No tag found for [ref:foo] @ ./cypress.json:1.
```

**System information:**
 - OS: macOS Catalina 10.15.7
 - Tagref version: 1.3.0

**Additional context**
Hi Stephan!

**Comment by stepchowfun** [comment]
Hi @sharmilajesupaul! Thanks for creating this bug! I found the root cause and should be able to release a fix by 3:30pm PT today. Keep your eyes peeled for v1.3.2! I'll close this issue when it's done.

**Comment by stepchowfun** [comment]
They say software timeline estimation is hard, but the new version of Tagref has arrived just on schedule!

Contrary to what I said in my previous message, you should install the newly released v1.3.3, not v1.3.2. The bug was indeed fixed in v1.3.2, but it had the unfortunate consequence of allowing Tagref to scan inside `.git` directories—normally not desirable. So v1.3.3 explicitly denylists `.git` (and `.hg` as well, for Mercurial users).

**Comment by sharmilajesupaul** [comment]
Just confirming that your fix works in v1.3.3! Thanks for the quick work on this! 

```                                                                                                                                                                                                                                                                                                 
~/airlab/repos/pineapple/frontend/playground-loop shar--bump-tagref* ⇡
❯ tagref
1 tag and 1 reference validated in 59 files.
```
